linode/manager
# chore: [M3-7474] Update storybook & add @babel/traverse resolution

## Description 📝
Main PR goal is to get address a vulnerability in `@babel/traverse` spotted by dependabot: https://github.com/linode/manager/security/dependabot/85

Tho storybook (package responsible for this vulnerability) does not address it in 7.6.10 it does not hurt to keep it up to date hence the bump. 

## Changes  🔄
- Add 7.23.3.`@babel/traverse` resolution 
- Update Storybook and dependent packages

## How to test 🧪

### Verification steps 
- Make sure storybook runs and build
  - `yarn storybook`
  - `yarn build-storybook`

## As an Author I have considered 🤔

*Check all that apply*
